src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -264,31 +288,30 @@ def _alloc_nompi(self, cud):
         Find a suitable set of cores and gpus *within a single node*.
 
         Input:
-        cud: Compute Unit description. Needs to specify at least one process,thread
-        or GPU process.
+        cud: Compute Unit description. Needs to specify at least one CPU 
+        process and one thread per CPU process, or GPU process.
```

Comment:
What happens if the `threads_per_process` are set to `0`?  :)

Reply:
My vote is to respond to the user with red lines! :P

I am happy to catch the case when user specifies 0 threads. My suggestion would be the following:
(i) we should document that these are not additional threads but the total number of threads. Increment the number of threads to 1 if user specifies it as 0.
or (ii) change the code base to reflect that these are additional threads and document the same.